docs/manual/index.md

```diff
@@ -16,6 +16,4 @@ This document assumes that you are generally familiar with Kubernetes. For more
 1. [Controlling Fault Domains](fault_domains.md)
 1. [Running with TLS](tls.md)
 1. [Backup](backup.md)
-1. [Technical Design](technical_design.md)
-1. [Debugging](debugging.md)
```

Comment:
Was it intentional to delete these? I think we still want them there.

Reply:
Nope 🤔 Adding this back.